.bashrc

```diff
@@ -55,10 +64,7 @@ alias alert='notify-send --urgency=low -i "$([ $? = 0 ] && echo terminal || echo
 # When running cd (current directory command) onto a git directory
 # show git status in bash
 # From: https://github.com/magicmonty/bash-git-prompt.git
-if [ -f ~/sources/clones/bash-git-prompt/gitprompt.sh ]; then
-    GIT_PROMPT_ONLY_IN_REPO=1
-    source ~/sources/clones/bash-git-prompt/gitprompt.sh
-fi
+source_if_exists ~/sources/clones/bash-git-prompt/gitprompt.sh export GIT_PROMPT_ONLY_IN_REPO=1
```

Comment:
Meaning has changed here (GIT_PROMPT_ONLY_IN_REPO is set after you have sourced). Don't know if it matters for your case.